.github/workflows/e2e-test.yml

```diff
@@ -44,6 +44,7 @@ jobs:
           kind create cluster --image kindest/node:v1.20.7@sha256:688fba5ce6b825be62a7c7fe1415b35da2bdfbb5a69227c499ea4cc0008661ca
           kubectl version
           kubectl cluster-info
+          kubectl describe nodes
```

Comment:
Why should we change this?

Reply:
not needed, was using it to debug earlier e2e issue. Will revert it